add-dataset/README.md

```diff
@@ -25,21 +25,42 @@ After going through this tutorial you will know how to:
 
 ### Add an Important Dataset{#lesson-one}
 
-#### Step 1. Get information about the dataset
+#### Step 1. Get information about a dataset
 
 Before you start you'll want to know a little bit about the dataset you would like to archive on Data Together. At a minimum this should include the **name** and **web address** or [**URL** ](https://en.wikipedia.org/wiki/URL) where the dataset can currently be accessed.
 
-Next, visit the Data Together [Add a Dataset](#) page
+An example would be EPA's National Greenhouse Gas Emission Inventory, currently located at https://catalog.data.gov/dataset/national-greenhouse-gas-emission-inventory. 
 
-<screenshot>
 
+#### Step 2. Register on Data Together
 
+To register on Data Together, visit https://archivers.co/signup
 
-#### Step 2. ??
+<img src="images/Screenshot-signup.png" width=250> <img src="images/screenshot-signup-1.png" width=250>
 
-### Register on Data Together (Optional){#lesson-two}
 
-1.
+#### Step 3. Archive a dataset on Data Together
+
+* Copy the URL of the dataset you would like to archive
```

Comment:
Maybe clarify that "copy the URL" means for people to copy it to their clipboard?